src/arkode/arkode_erkstep.c

```diff
@@ -1243,7 +1241,7 @@ int erkStep_RelaxDeltaE(ARKodeMem ark_mem, ARKRelaxJacFn relax_jac_fn,
     retval = relax_jac_fn(z_stage, J_relax, ark_mem->user_data);
     (*num_relax_jac_evals)++;
     if (retval < 0) { return ARK_RELAX_JAC_FAIL; }
-    if (retval > 0) { return ARK_RELAX_JAC_RECV; }
+    if (retval > 0) { return ARK_RELAX_BRENT; }
```

Comment:
As in `arkode_arkstep.c`, I don't see why this return was changed to `ARK_RELAX_BRENT`.

Reply:
I did not intentionally make this change, it must have been a merge issue.